src/app/core/config/views.ts

```diff
@@ -68,6 +69,7 @@ const views: Array<{
   { id: AppView.SharePhotosToken, component: SharePhotosView },
   { id: AppView.RedirectToApp, component: RedirectToAppView },
   { id: AppView.VerifyEmail, component: VerifyEmailView },
+  { id: AppView.RequestAccess, component: RequestAccess },
```

Comment:
Now it is accessible, hide it until the task is completed if this is just the UI.

Reply:
I have commented it so it is not available